src/Algebra/Graph/Undirected.hs

```diff
@@ -698,6 +700,25 @@ toSymmetricRelation :: Ord a => Graph a -> SR.Relation a
 toSymmetricRelation = foldg SR.empty SR.vertex SR.overlay SR.connect
 {-# INLINE toSymmetricRelation #-}
 
+-- | Complement of a graph.
+-- Complexity: /O(E+V)/ time, /O(E+V)/ memory where
```

Comment:
Please use `n` and `m` to denote the number of vertices and edges, respectively, as in the rest of the docs.

Reply:
Also, I don't think the complexity bounds are correct: you build up /O(n^2)/ edges in `edgeList (cliqueG g)`!